Applications/Resnet/jni/main.cpp

```diff
@@ -205,14 +205,14 @@ ModelHandle createResnet18() {
 }
 
 int trainData_cb(float **input, float **label, bool *last, void *user_data) {
-  auto data = reinterpret_cast<nntrainer::resnet::DataLoader *>(user_data);
+  auto data = reinterpret_cast<nntrainer::cifar::DataLoader *>(user_data);
```

Comment:
it might be better to use util namespace.

Reply:
[Fix} change namespace ```cifar``` to ```util```